tests/data/expected/main/malformed/parsed_empty_pointer.py

```
@@ -0,0 +1,2 @@
+class Empty(RootModel[Any]):
```

Ruff (0.15.21)

[error] Undefined name `RootModel`

(F821)